frontend/update-git-hash.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-
-cd "$(dirname "$0")"
```

Comment:
Not sure that we moved anywhere, so this is the only thing that is missing in a 1-1 conversion